Scholars Portal Journals.js

```diff
@@ -0,0 +1,172 @@
+{
+	"translatorID": "5ac0fd37-5578-4f82-8340-0e135b6336ee",
+	"label": "Scholars Portal Journals",
+	"creator": "Bartek Kawula",
+	"target": "https?://journals[1-2]\\.scholarsportal\\.info/",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2016-02-19 04:18:37"
+}
+
+function detectWeb(doc, url) { 
+	// see if saved list is toggled open
+	if (doc.getElementsByClassName('inner-wrap-open') [0]) {
+		return 'multiple'
+	} else {
+		if(url.indexOf("/search?q") != -1 || url.indexOf("/browse/") != -1){
+			return 'multiple'
+		}
+		else {
+			return 'journalArticle';
+		}
+	}
+}
+
+function doWeb(doc, url) {
+	var type = detectWeb(doc, url);
+	if(type == "multiple") {
+		var list = getItems(doc, url);
+		Zotero.selectItems(list, function(selectedItems) {
+			if(!selectedItems) return true;
+			var uri = [];
+			for(var i in selectedItems) {
+				uri.push(list[i].uri)
+			}
+			for(var i=0; i<uri.length; i++) {
+				var url = uri[i].replace(/(\/details)/g,"");
+				scrape(doc, url)
+			}
+		})
+	} else {
+		var a = url.indexOf("details");
+		var b = url.indexOf("xml");
+		url = url.substring(a+7,b+3);
+		scrape(doc, url)
+	}
+}
+
+function getItems(doc, url) {
+	if (doc.getElementsByClassName('inner-wrap-open') [0]) {
+		var titles = ZU.xpath(doc.getElementById('my-articles-list'), '//div[@class = "title"]/h3/a');
+		var items = {};
+		for (var i=0; i<titles.length; i++) { 
+			var title = ZU.trimInternal(titles[i].textContent);
+			var uri = ZU.trimInternal(titles[i].pathname);
+			items[i] = {'title':title,'uri':uri};
+		}
+		return items
+	} else {
+		if(url.indexOf("/browse") != -1){
+			var titles = ZU.xpath(doc, '//div/h4/a');
+		}
+		else {
+			var titles = ZU.xpath(doc.getElementById('result-list'), '//div[@class = "details"]/h3/a');
+		}
+		var items = {};
+		for (var i=0; i<titles.length; i++) { 
+			var title = titles[i].textContent;
+			var uri = titles[i].pathname;
+			items[i] = {'title':title,'uri':uri};
+		}
+		return items
+	}
+}
+
+function extras(doc, item) { 
+	var currentdate = new Date(); 
+	item.accessed = currentdate.toLocaleString();
+	return item
+}
+
+function scrape(doc, uri) { 
+		var risURL = 'http://'+doc.domain+'/ris?uri='+uri;
```

Comment:
use relative URLs `/ris?uri=...` and `encodeURIComponent()` the `uri`

Also, too many indents.